config/environments/development.rb

```diff
@@ -30,6 +30,7 @@
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
   config.active_storage.service = :local
+  config.active_storage.service = :test
```

Comment:
I think you can remove this one. `config.active_storage.service` has been defined to `local`

Reply:
So no need to test?

Reply:
yep